homeassistant/components/nice_go/coordinator.py

```diff
@@ -220,4 +247,38 @@ async def on_data(self, data: dict[str, Any]) -> None:
     async def on_connected(self) -> None:
         """Handle the websocket connection."""
         _LOGGER.debug("Connected to the websocket")
+        self.connected = True
+
         await self.api.subscribe(self.organization_id)
+
+        if not self.last_update_success:
+            self.async_set_updated_data(self.data)
+
+    async def on_connection_lost(self, data: dict[str, Exception]) -> None:
+        """Handle the websocket connection loss. Don't need to do much since the library will automatically reconnect."""
+        _LOGGER.debug("Connection lost to the websocket")
+        self.connected = False
+
+        # Give some time for reconnection
+        await asyncio.sleep(5)
```

Comment:
If you have tests for this, please make the 5 seconds a constant and patch it to be 0, otherwise the tests take too long

Reply:
You are right, I already asked this to be a const.

Reply:
Replaced with const, and replaced patching asyncio.sleep with patching the const